region/maridia/inner-green.json

```diff
@@ -669,6 +676,13 @@
                 "length": 0,
                 "requires": null
               }
+            ],
+            "canLeaveCharged": [
+              {
+                "usedTiles": 31,
+                "framesRemaining": 80,
+                "requires": [ "canUsePowerBombs" ]
```

Comment:
v

Reply:
Yeah if that glitch can't be fixed that might be the way to go

Reply:
It's regardless of the glitch actually. It's also just because you don't need to use PowerBombs ever again once Shaktool has dug the room out.